Runtime/Scripts/NetworkStream.cs

```diff
@@ -75,6 +77,27 @@ public bool isActive
 
         public IStreamSettings streamSettings { get { return m_StreamSettings; } }
 
+        /// <summary>
+        /// Manual override to use a specific stream recorder. This cannot be set during recording.
+        /// </summary>
+        public GameObject streamRecorderOverride
+        {
+            get => m_StreamRecorderOverride;
```

Comment:
Try to avoid C# 6 language features, which will break in the legacy runtime. We don't know what kind of projects people will try dropping this into. Easiest way to track them all down is to just switch the player settings back to 3.x and see what errors pop up. If it's just this and a few conveniences, I'd say fix them so we can use this in older projects.

Reply:
Good call. In my mind we were always upgrading the min version, and we still might, but I realize that most of the upgrades will be in the client repo.

Reply:
@mtschoen I'm getting a lot of:
`error CS1644: Feature `out variable declaration' cannot be used because it is not part of the C# 4.0 language specification` for MyDictionary.TryGetValue(key, out value)